util/drakeUtil.h

```diff
@@ -92,4 +96,37 @@ DLLEXPORT std::pair<Eigen::Vector3d, double> resolveCenterOfPressure(Eigen::Vect
 
 DLLEXPORT double *mxGetPrSafe(const mxArray *pobj);
 
+template<typename Derived>
+std::string to_string(Eigen::MatrixBase<Derived> & a)
+{
+	std::stringstream ss;
+	ss << a;
+	return ss.str();
+}
+
+
+template<typename Derived>
+void valuecheck( Eigen::MatrixBase<Derived>& a, Eigen::MatrixBase<Derived>& b, double tol, std::string error_msg)
```

Comment:
There should be two template arguments, one for a, one for b. Otherwise you can't e.g. compare a MatrixXd that was returned by some function to MatrixXd::Zero because they have different types.
Also, a and b should be const references.